Link fixes

kswiecicki opened the pull request:
Fixes were made based on the logs from https://github.com/pmem/pmem.github.io/issues/200.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/pmem/pmem.github.io/238)
<!-- Reviewable:end -->

lukaszstolarczuk reviewed (commented):
> Reviewed 51 of 51 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/238)* status: all files reviewed, 3 unresolved discussions (waiting on @kswiecicki)

___
*[`content/blog/2018/cpp-array.md` line 192 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/238#-N1sBM3Q8TGx7LEFO0Gh:-N1sBM3Q8TGx7LEFO0Gi:b-6reqnu) ([raw file](https://github.com/pmem/pmem.github.io/blob/2b05ff4dbc9bbe7fc4503f01ba5331eb62ccdba3/content/blog/2018/cpp-array.md#L192)):*
> ```Markdown
> so you can expect our containers collection to grow in the near future.
> 
> [cpp_array]: /libpmemobj-cpp/v1.5/doxygen/structpmem_1_1obj_1_1experimental_1_1array.html 'pmem::obj::array'
> ```

I think we should rather use `https://pmem.io/libpmemobj-cpp/master/doxygen/structpmem_1_1obj_1_1array.html` master docs (array is no longer experimental, that's why it was broken)

applies to all below
___
*[`content/blog/2018/release-1-5.md` line 249 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/238#-N1sBVV95-R4sSlkbUQ4:-N1sBVVAAUI-9wDgj_D7:b-avhirv) ([raw file](https://github.com/pmem/pmem.github.io/blob/2b05ff4dbc9bbe7fc4503f01ba5331eb62ccdba3/content/blog/2018/release-1-5.md#L249)):*
> ```Markdown
> We started the containers work easy, with `pmem::obj::array` that provides some
> convenience features for use with transactions. See our
> [doxygen](/libpmemobj-cpp/v1.5/doxygen/structpmem_1_1obj_1_1experimental_1_1array.html) documentation for more info.
> ```

.

`https://pmem.io/libpmemobj-cpp/master/doxygen/structpmem_1_1obj_1_1array.html`
___
*[`content/blog/2019/cpp-vector.md` line 295 at r1](https://reviewable.io/reviews/pmem/pmem.github.io/238#-N1sBf8X1aOiwSPIVDZ3:-N1sBf8X1aOiwSPIVDZ4:bmupk52) ([raw file](https://github.com/pmem/pmem.github.io/blob/2b05ff4dbc9bbe7fc4503f01ba5331eb62ccdba3/content/blog/2019/cpp-vector.md#L295)):*
> ```Markdown
> 
> [cpp_containers]: /blog/2018/11/c-persistent-containers/ 'blog post'
> [cpp_vector]: /libpmemobj-cpp/v1.6/doxygen/classpmem_1_1obj_1_1experimental_1_1vector.html 'pmem::obj:experimental:vector'
> ```

.

`https://pmem.io/libpmemobj-cpp/master/doxygen/classpmem_1_1obj_1_1vector.html`


<!-- Sent from Reviewable.io -->

pbalcer commented:
I merged without looking, sorry ;) please address Lukasz's comments in a separate PR. Thanks.

kswiecicki commented:
> I merged without looking, sorry ;) please address Lukasz's comments in a separate PR. Thanks.

Sure, @lukaszstolarczuk I've addressed your remarks here: https://github.com/pmem/pmem.github.io/pull/239.
